src/asm/lexer.c

```diff
@@ -921,6 +921,40 @@ void lexer_DumpStringExpansions(void)
 	free(stack);
 }
 
+/* Discards an block comment */
+static void discardBlockComment(void)
+{
+	dbgPrint("Discarding block comment\n");
+	bool block_ended = false;
+
+	for (;;) {
+		switch(nextChar()) {
```

Comment:
Code style mandates a space after `switch`; you can either [set checkpatch up yourself](https://github.com/gbdev/rgbds/blob/master/CONTRIBUTING.rst#contributing-code), or [look at the CI report](https://github.com/gbdev/rgbds/pull/607/checks) (it only fails on errors, but we prefer avoiding warnings and checks whenever reasonably possible)